pages/linux/lddd.md

```diff
@@ -0,0 +1,8 @@
+# lddd
+
+> Find broken library links on your system. This tool is only available on Archlinux.
+> More information: <https://man.archlinux.org/man/extra/devtools/lddd.1.en>.
+
+- Scan $PATH, /lib, /usr/lib, /usr/local/lib and /etc/ld.so.conf.d/*.conf to find out packages with broken library links:
```

Comment:
I think we don't need to list the directories here, that's why we have the more information link
```suggestion
- Scan directories for packages with broken library links:
```

Reply:
Agreed, Your description sounds better.